src/DDoSX/Entities/CdnRule.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace UKFast\SDK\DDoSX\Entities;
+
+use UKFast\SDK\Entity;
+
+/**
+ * @property string $id
+ * @property string $cacheControl
+ * @property string $type
+ * @property string $uri
+ * @property array $mimeTypes
+ * @property string cacheControlDuration
```

Comment:
nit: this isn't really a string, but an object

Reply:
How is this an object? 

#### APIo response
```
"cache_control": "Origin",
```